website/src/components/AddDevice.tsx

```diff
@@ -94,16 +99,27 @@ export const AddDevice = observer(class AddDevice extends React.Component<Props>
           <CardHeader title="Add A Device" />
           <CardContent>
             <form onSubmit={this.submit}>
-              <FormControl error={!!this.error} fullWidth>
+              <FormControl fullWidth>
                 <InputLabel htmlFor="device-name">Device Name</InputLabel>
                 <Input
                   id="device-name"
                   value={this.deviceName}
                   onChange={(event) => (this.deviceName = event.currentTarget.value)}
                   aria-describedby="device-name-text"
                 />
-                <FormHelperText id="device-name-text">{this.error}</FormHelperText>
+                <FormHelperText id="device-name-text">Any name to your liking</FormHelperText>
+              </FormControl>
+              <FormControl fullWidth>
+                <InputLabel htmlFor="device-publickey">Device Public Key</InputLabel>
```

Comment:
I'd point out that this is optional here, maybe `Device Public Key (optional)`?

Reply:
Yes, I noticed this missing piece of information as well and added it already here https://github.com/freifunkMUC/wg-access-server/pull/308/files#diff-5dc47881e3984985725c6d2224baf9b5d89e5415a7a9a55a1b642d4919882dafR142

Reply:
Added it in the newest commit.